webapp/src/container.tsx

```diff
@@ -439,7 +439,9 @@ export class SideDocs extends data.Component<SideDocsProps, SideDocsState> {
 
     setMarkdown(md: string) {
         const docsUrl = pxt.webConfig.docsUrl || '/--docs';
-        const mode = this.props.parent.isBlocksEditor() ? "blocks" : "js";
+        // always render blocks by default when sending custom markdown
+        // to side bar
+        const mode = "blocks" // this.props.parent.isBlocksEditor() ? "blocks" : "js";
```

Comment:
How come?

Reply:
The way to get through Readme.md is through the explorer, which de-facto sets the editor to javascript. However, the setting markdown feature is mostly useful to preview readme.md... which mostly is about seeing blocks.